app/v1/validation/revokeReportingCompany/RevokeReportingCompanyValidator.scala

```diff
@@ -45,7 +45,7 @@ trait RevokeReportingCompanyValidator extends BaseValidation {
     }
   }
 
-  private def validateUltimateParentCompany(implicit path: JsPath): ValidationResult[Option[UltimateParentModel]] = {
+  private def validateUltimateParentCompany: ValidationResult[Option[UltimateParentModel]] = {
```

Comment:
Why is the path not needed here anymore?

Reply:
It wasn't used in the method

Reply:
After checking it out, I see the path is defined in the validation case class, compared with the other ones that use an implicit for it